include/power/power_state.h

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2020 Intel corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_POWER_POWER_STATE_H_
+#define ZEPHYR_INCLUDE_POWER_POWER_STATE_H_
+
+#include <sys/util.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#define PM_STATE_RUNTIME_ACTIVE  BIT(0)
+#define PM_STATE_RUNTIME_IDLE    BIT(1)
+#define PM_STATE_SUSPEND_TO_IDLE BIT(2)
+#define PM_STATE_STANDBY         BIT(3)
+#define PM_STATE_SUSPEND_TO_RAM  BIT(4)
+#define PM_STATE_SUSPEND_TO_DISK BIT(5)
+#define PM_STATE_SOFT_OFF        BIT(6)
+
+#define PM_STATE_BIT_MASK        0x7F
```

Comment:
`BIT_MASK(7)` would be more clearly tied to the number of states identified by `BIT(n)` above.